great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -134,6 +134,7 @@
 T = TypeVar("T", dict, list, str)
 
 
+@public_api
```

Comment:
It makes me a bit sad that we have to make an abstract class part of our public api to get our docs to render, but so be it.